Lib/fontTools/ttLib/tables/otBase.py

```diff
@@ -1150,7 +1150,7 @@ def fromXML(self, name, attrs, content, font):
         except KeyError:
             raise  # XXX on KeyError, raise nice error
         value = conv.xmlRead(attrs, content, font)
-        if conv.repeat:
+        if conv.repeat is not None:
```

Comment:
What prompted this? can repeat ever be 0?

Reply:
I'm (ab)using a conv.repeat of `""` to mean it's a list that is handled by manual code for decompile/compile purposes.

Reply:
perhaps that warrants a comment..